.github/workflows/_s3_publish.yml

```
@@ -0,0 +1,287 @@
+        id: manifest
+        run: |
+          S3_PREFIX="${{ steps.upload.outputs.s3_prefix }}"
+          PUBLISH_TO_LATEST="${{ steps.upload.outputs.publish_to_latest }}"
+          GIT_VERSION="${{ steps.version.outputs.git_version }}"
+          BUILD_DATE="${{ steps.version.outputs.build_date }}"
+          
+          echo "Creating manifest for $GIT_VERSION (S3 prefix: $S3_PREFIX)"
+          
+          # Create JSON manifest that points to versioned paths
+          cat > latest-manifest.json << EOF
+  {
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 161: could not find expected ':'

(syntax-check)